configuration/messageproperties/messages_zl_en.properties

```diff
@@ -29,8 +29,8 @@ zl.status.hivTb.inhCompletionDate=INH Completion Date
 zl.status.hivTb.inhComplete=Complete
 zl.status.hivTb.inhNotComplete=Not Complete
 zl.status.hivTb.inhNotStarted=Uninitiated
-zl.status.regimen.lastPrescribedMedication=Last prescribed medication
-zl.status.regimen.lastDispensedMedication=Last dispensed medication
+zl.status.regimen.lastPrescribedMedication=Last prescribed ART
```

Comment:
The message code should be similar -- zl.lastPrescribedART and zl.lastDispensedART.